server/api/api.go

```diff
@@ -34,6 +34,8 @@ const (
 	ErrorNoTeamMessage = "No team"
 )
 
+var errEmpty = errors.New("")
```

Comment:
This looks really wrong, I would like to now why something is failing, In this case I would say something like "Not supported API in standalone mode" or something like that.

Reply:
Done.